src/basic/cgroup-util.h

```diff
@@ -60,6 +60,15 @@ typedef enum CGroupMask {
         _CGROUP_MASK_ALL = CGROUP_CONTROLLER_TO_MASK(_CGROUP_CONTROLLER_MAX) - 1
 } CGroupMask;
 
+static inline CGroupMask CGROUP_MASK_EXTEND_JOINED(CGroupMask mask) {
+        /* We always mount "cpu" and "cpuacct" in the same hierarchy. Hence, when one bit is set also set the other */
+
+        if (mask & (CGROUP_MASK_CPU|CGROUP_MASK_CPUACCT))
+                mask |= (CGROUP_MASK_CPU|CGROUP_MASK_CPUACCT);
+
+        return mask;
```

Comment:
Shouldn't this also cover net_cls, net_prio?

Reply:
We don't have any management or mask within systemd for `net_cls` or `net_prio` in cgroup v1, so I don't think we could do anything here. The fact that we mount them together is largely for external convenience, as far as I can tell.

Reply:
Yes, we only mount those controllers, we don#t ever make use of them, and this is not going to change as they have no future on cgroupsv2.